### ISSUE TITLE
Interoperability Integrated Text Annotation and Linked Open Data

### ISSUE THREAD

**Issue by rlzijdeman** [CLARIAH/usecases#45]
https://github.com/CLARIAH/usecases/blob/master/cases/folia-lod.md

**Comment by roelandordelman** [comment]
This use case seems more a technical use case than a scholarly use case, at least the description of the research requirement (related to the primitive "annotation") is on a meta level, being "as free as possible to move between different ecosystems and use the tooling that is designed for these ecosystems". The main goal that emerges from the use case is improving the operability of Folia by developing converters and mappings. Would it be possible to elaborate on the necessity of this work for scholars? What kind of scholarly use cases are involded here?

**Comment by proycon** [comment]
Yes, you're right. This is a fairly technical and generic use case. It was written from the perspective of how we can
tie the things we have and have developed over the past decade (FoLiA and various FoLiA-aware tools) to the increasing
trend of using linked open data.

I don't have a specific scholarly use-case in mind. The goal is to ensure that researchers working with various
FoLiA-based corpora/resources/tools, in whatever shape or form, can find easier connection to the wider CLARIAH
infrastructure as it develops, especially if that moves largely in a LOD direction. The necessity from my perspective is
to prevent having completely disconnected ecosystems (as is the case now).
